apps/communications/contacts/style/contacts.css

```diff
@@ -426,7 +425,7 @@ gaia-header [data-icon] {
 #import-settings .view-body-inner,
 #view-contact-form .view-body-inner,
 #ice-settings .view-body-inner {
-  background: #f4f4f4;
+  background: #ffffff;
```

Comment:
Why are you making minor color adjustments?  I thought we only had wireframes. Are there already visual design specs that you're following here?